.github/workflows/e2e-nvidia-t4-x1.yml

```diff
@@ -124,12 +124,13 @@ jobs:
           # install with Torch and build dependencies installed
           python3.11 -m pip install -v packaging wheel setuptools-scm
           python3.11 -m pip install -v .[cuda]
-        
+
```

Comment:
All `install ilab` steps should use `PYTHON=python3.11 ./scripts/install-ilab-with-cuda.sh` from `instructlab/instructlab` repo, otherwise these `pip install` commands won't use the correct constraints. (Plus it simplifies the workflow.)